swarm/api/routes/wisdom.py

```
@@ -505,6 +507,12 @@ async def get_wisdom_content(
         404: Artifact not found.
         304: Not modified (if ETag matches).
     """
+    try:
+        validate_path_component(run_id, "run_id")
+        validate_path_component(artifact_name, "artifact_name")
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.15.4)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)